src/main/java/org/jabref/logic/integrity/BracesCorrector.java

```diff
@@ -0,0 +1,26 @@
+package org.jabref.logic.integrity;
+
+public class BracesCorrector {
+
+    public static String apply(String s) {
+        if (s == null) {
+            return null;
+        } else {
+            String addedBraces = s;
+            String c = addedBraces.replaceAll("\\\\\\{", "").replaceAll("\\\\\\}", "");
```

Comment:
I think it would make sense to extract the regex to a Pattern:
https://stackoverflow.com/questions/1466959/string-replaceall-vs-matcher-replaceall-performance-differences